pinot-query-runtime/src/test/resources/queries/JoinQuery.json

```diff
@@ -0,0 +1,200 @@
+{
+  "basic_join_queries": {
```

Comment:
hmm, I thought I left this somewhere but I asked for:
- tests involving joins on different types
- tests with joins that have functions in the join condition
- joins without join conditions (`SELECT * FROM a,b`)

Reply:
Sound good will add.